packages/modules/create-imodel/src/components/update-imodel/UpdateIModel.test.tsx

```diff
@@ -82,6 +83,76 @@ describe("UpdateIModel", () => {
     );
   });
 
+  it("should enable update when extent is removed", async () => {
+    const successMock = jest.fn();
+    toaster.positive = jest.fn();
+
+    const { getByText, rerender } = render(
+      <UpdateIModel
+        accessToken="dd"
+        imodelId="de47c5ad-5657-42b8-a2bc-f2b8bf84cd4b"
+        onSuccess={successMock}
+        apiOverrides={{ serverEnvironmentPrefix: "dev" }}
+        initialIModel={{
+          name: "Initial name",
+          description: "Initial description",
+          extent: {
+            southWest: { latitude: 1, longitude: 2 },
+            northEast: { latitude: 3, longitude: 4 },
+          },
+        }}
+      />
+    );
+
+    const updateButton = getByText("Update");
+    expect(updateButton.closest("button")?.hasAttribute("disabled")).toBe(true);
+
+    rerender(
+      <UpdateIModel
+        accessToken="dd"
+        imodelId="de47c5ad-5657-42b8-a2bc-f2b8bf84cd4b"
+        onSuccess={successMock}
+        apiOverrides={{ serverEnvironmentPrefix: "dev" }}
+        initialIModel={{
+          name: "Initial name",
+          description: "Initial description",
+          extent: {
+            southWest: { latitude: 1, longitude: 2 },
+            northEast: { latitude: 3, longitude: 4 },
+          },
+        }}
+        extent={(null as unknown) as iModelExtent}
```

Comment:
In order to prevent this cast, I think you better change `extent` type: `extent?: iModelExtent | null`